app/styleguides/_default/components/subheader.css

```diff
@@ -6,3 +6,43 @@ $helvetica-stack: Helvetica, sans-serif;
   font: bold 22px $helvetica-stack;
   margin: 0 0 1em;
 }
+
+.ui-radio-group.is-vertical .ui-radio-group__radios {
+  flex-direction: row !important;
+}
+
+.h4 {
+  font-size: 22px;
+  line-height: 1.09;
+}
+
+.h3 {
+  font-size: 28px;
+  line-height: 1.09;
+  & > .link {
+    padding: 3px 0px 0px 5px;
+  }
+}
+
+.h2 {
+  font-size: 38px;
+  line-height: 1.05;
+  & > .link {
+    padding: 0px 0px 0px 5px;
+  }
+}
+
+.subheader:hover {
+  & .link {
+    opacity: 1;
+  }
+}
+
+.link {
```

Comment:
If we move all of these styles to be nested under `.subheader { ... }` then we'll make sure styles from components don't bleed into each other as we add new components.